src/elements/content-sidebar/activity-feed/comment/Comment.js

```diff
@@ -51,12 +51,12 @@ type Props = {
     tagged_message: string,
     translatedTaggedMessage?: string,
     translations?: Translations,
-    type?: typeof COMMENT_TYPE_DEFAULT | typeof COMMENT_TYPE_TASK,
+    type: typeof COMMENT_TYPE_DEFAULT | typeof COMMENT_TYPE_TASK,
     userHeadlineRenderer?: React.Node => React.Element<typeof FormattedMessage>,
 };
 
 type State = {
-    isConfirming?: boolean,
+    isConfirming: boolean,
```

Comment:
I am guessing the other two also dont need ? since its set in state

Reply:
Fixed, can you approve again please @priyajeet?